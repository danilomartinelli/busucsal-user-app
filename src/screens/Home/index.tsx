import * as React from "react";
import { SafeAreaView } from "react-native";
import { NavigationScreenProp } from "react-navigation";
import BusRoutesItem from "../../components/BusRoutes/BusRoutesItem";
import Header from "../../components/Header";

import styles from "../../resources/styles";
import { ProgressBar } from "react-native-paper";
import ActivityIndicator from "../../components/ActivityIndicator";

export interface IProps {
  navigation?: NavigationScreenProp<{}>;
}

export interface IState {
  isLoading: boolean;
}

class Home extends React.Component<IProps, IState> {
  constructor(props: IProps) {
    super(props);
    this.state = {
      isLoading: true,
    };
  }

  public render() {
    const { navigation } = this.props;
    const { isLoading } = this.state;

    return (
      <SafeAreaView style={styles.Flex}>
        <Header title="Roteiros" subtitle="Próximos horários" />
        <ActivityIndicator size="large" color="blue" />
      </SafeAreaView>
    );
  }
}
export default Home;
