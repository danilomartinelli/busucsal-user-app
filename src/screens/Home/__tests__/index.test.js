import React from "react";
import { shallow } from "enzyme";
import Home from "..";

describe("Home Screen", () => {
  describe("render", () => {
    it("should render properly", () => {
      const wrapper = shallow(<Home />);
      expect(wrapper).toMatchSnapshot();
    });
  });

  test.todo("New Tests for new behavior");
});
