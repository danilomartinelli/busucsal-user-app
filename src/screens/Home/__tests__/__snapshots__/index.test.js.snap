// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen render should render properly 1`] = `ShallowWrapper {}`;
